### ISSUE TITLE
Add arch to Dive Image info

### ISSUE THREAD

**Issue by valnoxy** [valnoxy/Dive#4]

**Comment by valnoxy** [comment]
not possible as image description doesn't contain any needed information

**Comment by valnoxy** [comment]
Found it!

```/WIM/IMAGE/WINDOWS/ARCH```

Syntax:
```xml
<ARCH>0</ARCH> = Architecture x86
<ARCH>9</ARCH> = Architecture amd64
```

**Comment by valnoxy** [comment]
fixed with commit [60787a2](https://github.com/valnoxy/Dive/commit/60787a2595a6fe70f99ff0b8bee1a0a482bf43a6)
